glad/generator/rust/templates/impl.rs

```diff
@@ -54,20 +67,31 @@ pub mod enumerations {
 }
 
 pub mod functions {
-    #![allow(non_snake_case, unused_variables, dead_code)]
+    #![allow({{ 'non_snake_case, ' if not options.mx }}unused_variables, dead_code)]
```

Comment:
Same here and all the other's just allow what you have to allow

Reply:
Should be fixed in https://github.com/Dav1dde/glad/pull/256/commits/3c1d7934900137cecf580409a64b890e30e97a1c.